packages/opentelemetry-exporter-zipkin/README.md

```diff
@@ -8,18 +8,58 @@ OpenTelemetry Zipkin Trace Exporter allows the user to send collected traces to
 
 [Zipkin](https://zipkin.io/) is a distributed tracing system. It helps gather timing data needed to troubleshoot latency problems in microservice architectures. It manages both the collection and lookup of this data.
 
+## Prerequisites
```

Comment:
Are we sure that we want to cover setting up zipkin here?
I think that a link to zipkin.io would be less redundant and would not mean that we have to update the docs once they change the url to quickstart.zipkin.io.

Reply:
I thought this is easier for users, instead of jumping on multiple links. Having said that, I am ok to remove this section let me know.

btw, we did it same for [Jaeger exporter](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-exporter-jaeger#prerequisites).

Reply:
For me, this only covers a very limited use case: Someone that just wants to try it out and doesn't know zipkin.

Anyone else with an existing Zipkin deployment won't need it and someone that needs the quickstart to get the docker version running to try it out should ideally head over to zipkin to learn more.

Reply:
Ok will remove this section.